cmd/config/metrics-profiles/metrics.yml

```diff
@@ -37,6 +37,37 @@
 - query: irate(container_runtime_crio_operations_latency_microseconds{operation_type="network_setup_pod"}[2m]) > 0
   metricName: containerNetworkSetupLatency
 
+# Node metrics: Network
+- query: sum by (instance) ( irate(node_network_receive_bytes_total[5m]) + irate(node_network_transmit_bytes_total[5m]))
+  metricName: nodeNetworkTotal
+
+- query: sum by (instance) ( irate(node_network_receive_bytes_total[5m]))
+  metricName: nodeRxNetwork
+
+- query: sum by (instance) ( irate(node_network_transmit_bytes_total[5m]))
+  metricName: nodeTxNetwork
+
+- query: sum by (instance) ( irate(node_network_drop_total[5m]) )
+  metricName: nodeNetworkDropTotal
+
+- query: sum by (instance) ( irate(node_network_errs_total[5m]) )
+  metricName: nodeNetworkErrTotal
+
+- query: sum by (instance, device) (irate(node_network_receive_bytes_total[5m]) + irate(node_network_transmit_bytes_total[5m]))
+  metricName: nodeNetworkDeviceTotal
+
+- query: sum by (instance, device) (irate(node_network_receive_bytes_total[5m]))
+  metricName: nodeRxNetworkDevice
+
+- query: sum by (instance, device) (irate(node_network_transmit_bytes_total[5m]))
+  metricName: nodeTxNetworkDevice
+
+- query: sum by (instance, device) ( irate(node_network_drop_total[5m]) )
```

Comment:
The metric `node_network_drop_total` doesn't exist, this metric is broken down in:

- node_network_receive_drop_total
- node_network_transmit_drop_total

we also should rule out some system devices and null datapoints. i.e:

```shell
rate(node_network_transmit_drop_total{device!~"lo|ovs-system"}[2m]) > 0
```